pkgs/applications/audio/eartag/default.nix

```
@@ -0,0 +1,76 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook4
+, libadwaita
+, gettext
+, glib
+, desktop-file-utils
+, appstream-glib
+, gtk4
+, librsvg
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "eartag";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "knuxify";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-vNAKGwcvkZ3YdrA0uYTnlyta2az/bmjgCE/6e9NADT0=";
+  };
+
+  postPatch = ''
+    patchShebangs ./build-aux/meson/postinstall.py
+    chmod +x ./build-aux/meson/postinstall.py
```

```suggestion
    chmod +x ./build-aux/meson/postinstall.py
    patchShebangs ./build-aux/meson/postinstall.py
```
Isn't patchSheabangs requiring +x bit? Otherwise we might as well drop it.

---

Good catch! I didn't know that requires files to be executable.